REVIEWING.md

```diff
@@ -15,6 +15,10 @@ should be used for all pull requests:
 - [ ] Is there a risk of performance regressions? If so, have run the [Vector test harness](https://github.com/timberio/vector-test-harness)? (see [Performance Testing](#performance-testing))
 - [ ] Should documentation be adjusted to reflect any of these changes? (see [Documentation](#documentation))
 
+## Expectations
+
+We endeavour to review all PRs within 48 business hours (Monday to Friday - broadly Eastern Time) of submission.
```

Comment:
Maybe "within 48 hours on weekdays"? It's hard to pick a phraseology here that isn't loaded with (North) American assumptions.

Reply:
I mentioned this [in Slack](https://timberio.slack.com/archives/C0L38SXDX/p1598466725067400?thread_ts=1598466543.066900&cid=C0L38SXDX) so apologies for redundancy, but I think "2 business days" is safest and accounts for weekends.

We'd typically say "2 working days" in the UK, but that feels a bit local.

Reply:
I like "2 working days".